examples/eth-dm/src/waku.ts

```diff
@@ -9,14 +9,7 @@ export const PublicKeyContentTopic = '/eth-dm/1/public-key/proto';
 export const DirectMessageContentTopic = '/eth-dm/1/direct-message/proto';
 
 export async function initWaku(): Promise<Waku> {
-  const waku = await Waku.create({});
-
-  // Dial all nodes it can find
-  getStatusFleetNodes().then((nodes) => {
-    nodes.forEach((addr) => {
-      waku.dial(addr);
-    });
-  });
+  const waku = await Waku.create({ bootstrap: true });
```

Comment:
A developer could decide to pass `false` or simply not pass the option if they want to manually dial specific nodes once Waku is started, instead of having `libp2p-bootstrap` add the bootstrap nodes to the peer manager and let the peer manager auto dial in the background.

As we do not have users yet it is hard to know the usage patterns.

Reply:
Please note a developer can also pass an array of multiaddresses to bootstrap to or a function that returns multiaddresses, e.g.:

https://github.com/status-im/js-waku/blob/7160137bdbeac6b46892f6b5cd7ee6f15a4caa06/src/lib/waku.spec.ts#L48

https://github.com/status-im/js-waku/blob/7160137bdbeac6b46892f6b5cd7ee6f15a4caa06/src/lib/waku.spec.ts#L82